src/databricks/sql/thrift_backend.py

```diff
@@ -185,6 +189,11 @@ def __init__(
         self.force_dangerous_codes = kwargs.get("_retry_dangerous_codes", [])
 
         additional_transport_args = {}
+        _max_redirects: int = kwargs.get("_retry_max_redirects", {})
```

Comment:
Why is there this multilayer structure?  I would expect _retry_max_redirects to be an int.

Reply:
Good catch! This was an artefact from development. Fixed in 645a0304e86421ea1141d5f61be23b69c615f84c